Descent3/config.cpp

```diff
@@ -1049,9 +946,6 @@ struct sound_menu {
   };
 };
 
-#ifdef MACINTOSH
-#pragma mark toggles --
```

Comment:
`#pragma mark` macros are supported by compilers. You could probably remove the `MACINTOSH` check and just have it out bare.